database/migrations/2024_03_11_000000_add_origin_and_destination_id_to_trips.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+/**
+ * "SQLite doesn't support multiple calls to dropColumn / renameColumn in a single modification."
+ * -> so small steps here. (╯°□°)╯︵ ┻━┻
+ */
+return new class extends Migration
+{
+    public function up(): void {
+        Schema::table('hafas_trips', static function(Blueprint $table) {
+            //Columns need to be nullable for the migration
+            $table->unsignedBigInteger('origin_id')->nullable()->after('origin');
+            $table->unsignedBigInteger('destination_id')->nullable()->after('destination');
+
+            $table->foreign('origin_id')->references('id')->on('train_stations');
+            $table->foreign('destination_id')->references('id')->on('train_stations');
+        });
+
+        DB::table('hafas_trips')->update([
```

Comment:
Won't this be like... suuuuper slow? We currently have over 2 million trips. These 4 lines should equate to at least ~6 Million Queries.